### PR TITLE
ensure we correctly cleanup the context

### PR DESCRIPTION
**Pull request by stefanpenner**
- [x] pending confirmation in my setup

### CODE REVIEW

File: lib/ember-test-helpers/test-context.js [hunk 1]
```diff
@@ -7,3 +7,7 @@ export function setContext(context) {
 export function getContext() {
   return __test_context__;
 }
+
+export function unsetContext() {
```
Comment (rwjblue) [inline lib/ember-test-helpers/test-context.js]: Why do we need another method here? Could you call 'setContext(undefined)'?

Comment (stefanpenner) [inline lib/ember-test-helpers/test-context.js]: wanted it to be very obvious that It was specifically to unset, not just an overloaded the existing set.

Comment (rwjblue) [inline lib/ember-test-helpers/test-context.js]: Gotcha, :+1:

**Comment by chadhietala** [comment]
@stefanpenner :+1: Thanks for dedicating your Mondays to fixing leaks.

**Comment by stefanpenner** [comment]
Maybe I should spend Tuesday's learning how to spell:p

**Comment by stefanpenner** [comment]
@rwjblue i think this is good to go.

**Comment by rwjblue** [comment]
Released in:

* ember-test-helpers@0.5.5
* ember-qunit@0.4.5

**Pull request merged**